ajax/libs/mui/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "mui",
+  "filename": "js/mui.min.js",
+  "description": "A effective front-end framework closest to the native app",
```

Comment:
@PeterDaveHello How about `A effective front-end framework whose performance is closest to the native app`?